rye/src/config.rs

```diff
@@ -200,6 +200,15 @@ impl Config {
             .unwrap_or(true)
     }
 
+    /// Make the `.venv` to use the system site packages
+    pub fn venv_system_site_packages(&self) -> bool {
+        self.doc
+            .get("behavior")
+            .and_then(|x| x.get("venv-system-site-packages"))
+            .and_then(|x| x.as_bool())
+            .unwrap_or(false)
+    }
```

Comment:
The setting seems more flexible if it was a project specific one, what do you think?